concourse/enumerator.py

```diff
@@ -24,20 +24,36 @@
 from github3.exceptions import NotFoundError
 
 from ci.util import (
-    parse_yaml_file,
-    load_yaml,
-    info,
+    LintingError,
     fail,
-    verbose,
+    info,
+    lint_yaml,
+    load_yaml,
     merge_dicts,
     not_empty,
     not_none,
+    parse_yaml_file,
+    warning,
+    verbose,
 )
 from model.base import ModelBase, NamedModelElement
 from concourse.factory import RawPipelineDefinitionDescriptor
 import ccc.github
 
 
+REPLICATION_LINTING_CFG = {
+    'extends': 'relaxed',
+    'rules': {
+        'key-duplicates': {
+            'level': 'error',
```

Comment:
is this not the default?
```d051236 ~/src/cc-utils$ yamllint -drelaxed x.yaml                        
x.yaml
  2:1       error    duplication of key "a" in mapping  (key-duplicates)

d051236 ~/src/cc-utils$ cat x.a                                          
cat: x.a: No such file or directory
d051236 ~/src/cc-utils$ cat x.yaml                                       
a: 42
a: 10
```

Reply:
It is. Not a good day to read the docs, I guess. Removed